### PR TITLE
feat: read summary events

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

stacked on top of https://github.com/PostHog/posthog/pull/15245

very draft

## Changes

* adds a new API query param
* behind a flag
* which switches to listing sessions from the new session replay summary table

### still to-do

- [ ] merge #15245 
- [ ] tidy up the SessionList inheritance
- [ ] fix or restrict hogql properties

### for follow-up work

- [ ] not using a custom filter picker
- [ ] using a query instead of a filter

## How did you test this code?

developer tests and trying it locally

### CODE REVIEW

**Comment by pauldambra** [comment]
@benjackwhite this is a bit "force the straightest path through the code that's there" of an approach but it works 😓 

I'm tempted to spend effort deleting the inheritance rather than working around it, and seeing how we can hog-qlify the query instead of accounting for every possible filter change ourselves

File: frontend/src/scenes/session-recordings/playlist/sessionRecordingsListLogic.ts [hunk 1]
```diff
@@ -159,7 +159,11 @@ export const sessionRecordingsListLogic = kea<sessionRecordingsListLogicType>([
                         ...values.filters,
                         person_uuid: props.personUUID ?? '',
                         limit: RECORDINGS_LIMIT,
-                        version: values.featureFlags[FEATURE_FLAGS.RECORDINGS_LIST_V2] ? '2' : '1',
+                        version: values.featureFlags[FEATURE_FLAGS.SESSION_RECORDING_SUMMARY_LISTING]
```
Comment (benjackwhite) [inline frontend/src/scenes/session-recordings/playlist/sessionRecordingsListLogic.ts:162]: lol

File: posthog/queries/session_recordings/session_recording_list_from_replay_summary.py [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+           min(min_first_timestamp) as start_time,
+           max(max_last_timestamp) as end_time,
+           dateDiff('SECOND', start_time, end_time) as duration,
+           argMinMerge(first_url) as first_url,
+           sum(click_count),
+           sum(keypress_count),
+           sum(mouse_activity_count),
+           round((sum(active_milliseconds)/1000)/duration, 2) as active_time
+        FROM session_replay_events
+        PREWHERE team_id = %(team_id)s
```
Comment (benjackwhite) [inline posthog/queries/session_recordings/session_recording_list_from_replay_summary.py:101]: Feel like there is still some DRYing to do here but not blocking at all

**Review by benjackwhite — COMMENTED** [review]
First pass looks good overall but I didn't do too much of a deep dive

File: posthog/queries/session_recordings/test/__snapshots__/test_session_recording_list_from_session_replay.ambr
Comment (pauldambra) [inline posthog/queries/session_recordings/test/__snapshots__/test_session_recording_list_from_session_replay.ambr:1]: note to self when I come back around to this... there should be more queries being picked up here 🤔

**Pull request merged**